README.md

```diff
@@ -1,77 +1,122 @@
 
 
 ## ✏️ Design
 
 ### 📏 Entity-Relationship Diagram
 ![ERD Diagram](https://github.com/Zhi-704/c11-apollo-bandcamp-tracker/blob/main/diagrams/ERD.png)
+
+This diagram explicitly shows how the data extracted from the API is stored in the database. We have decided to use a **3NF RD** for our database due to the requirements of the tracker. As we care about various sources of data such as countries, tags, and artists, other types of schema such as STAR Schema would not be suitable for our aims as we need to query multiple tables. 
+
 ### 📐 Architecture Diagram
 ![Architecture Diagram](https://github.com/Zhi-704/c11-apollo-bandcamp-tracker/blob/main/diagrams/Architecture_Diagram.png)
 
+This is a diagram that shows how every different AWS services work and interact with each other in order to complete the requirements of the tracker.
+
+#### **IMPORTANT**
+ >Information regarding each section of the AD will not be found here. Instead, **clicking on** the bullet points below will send you to the relative **README** located in the **directories** of the section that you would like to look at. There you can find out all the information that you need.
+
+
+- [**ETL Pipeline**](./pipeline/README.md)
+
+- [**RDS**](./schema/README.md)
+
+- [**PDF Report**](./pdf_report/README.md)
+
+- [**Notifications**](./notifications/README.md)
+
+- [**Dashboard**](./dashboard/README.md)
+
+
+
+
 ## ✅ Getting Started
 
 ### 💿 Installations
-The following languages/softwares are required for this project. Things assigned as optional are only required if you desire to host this system on the cloud.
+The following languages/softwares are **required** for this project. Things assigned as **optional** are only required if you desire to host this tracker on the cloud.
 - Python
 - Bash
 - Docker (Optional)
 
 ### ❗️ Dependencies
-There are various folders for each part of the project. In order to run the scripts in each folder, you will need to install the required libraries. This can be done using the code provided below:
+There are various **directories** for each part of the project. In order to run the scripts in each folder, you will need to install the **required libraries**. This can be done using the code provided below:
 ```
 python3 -m venv .venv
 source .venv/bin/activate
 pip install -r requirements.txt
 ```
 
 ### 🏃‍♂️‍➡️ Running the scripts
-All the scripts only require basic commands to be executed. Different commands are used depending on the software. Ensure that you are in the directory of the file you want to run before using these commands.
+All the scripts only require **basic** commands to be executed. Different commands are used **depending on the software**. Ensure that you are in the directory of the file you want to run before using these commands.
 ```
 # Python
 python3 "script.py"
 
 # Bash
 bash "script.sh"
 
+# Terraform
+terraform init
+terraform apply
+yes
+
 # Docker
 docker build -t "image"
 docker run --env-file .env -t "image: tag"
 ```
```

Comment:
This should not be one big code block, there should be a code block for EACH sub header. It would look better that way.